bitcoin-core-dev-tech/2017-09-06-signature-aggregation.es.md

```diff
@@ -0,0 +1,78 @@
+
+¿Afecta lo de la multisig al formato de la firma? La agregación de claves no lo cambia. Sólo resuelve pero es compatible con. Bellare-Neven se reduce a Schnorr para una sola clave y si haces alguna de las agregaciones de claves, puedes usar esas claves en Bellare-Neven aunque sea agregada. Digamos que hay una multisig y Bryan tiene la clave y yo tengo su clave y mi propia clave, y él ha verificado mi clave también, entonces afirmamos que P es la suma de nuestras claves. Le decimos a la gente que P es nuestra clave, la gente paga a P. Y la gente también ha pagado a Greg que es una sola clave. Así que ahora hay Pbryan + Psipa = Pbs. Y Pgre. Podemos crear un multisig de Bellare-Neven que es verificable... Alguien paga a Pbs y alguien ha pagado a Pgreg y podemos construir conjuntamente una única transacción que gaste Pgreg y la moneda unida con todo lo que quieras. Los tres podemos colaborar para construir esta transacción, pero lo único que ve el mundo es (sig, Pbs, Pgreg, M) donde M es el mensaje. Y Greg/Pieter no podría averiguar un valor para engañar a Bryan, porque la firma no sería válida debido a las propiedades de Bellare-Neven multisig. Puedes firmar un mensaje para demostrar que conoces el registro discreto. El problema de la cancelación podría existir... Pieter podría convencer a Bryan de no seguir el protocolo correcto, y hacer un ataque de cancelación. El esquema anterior resolvería esto, pero no sabemos si es tan seguro como el esquema Bellare-Neven. Los participantes pueden firmar con su única clave que quieren agregar, para demostrar que no tienen trucos de cancelación, pero es interactivo. Hay una técnica de delineación, pero sin una buena prueba de seguridad. Se puede pre-firmar un mensaje, si se publica la firma en algún lugar, por lo que no es necesario que sea interactivo.
+
+La agregación de claves es que entre los cosignatarios usted mismo firma su clave pública, lo que demuestra que tiene la clave privada. Este es el comportamiento de la cartera para multisig. Es compatible con las cosas de la firma agregada, que son reglas de la red.
+
+Andytoshi ha implementado Bellare-Neven multisig. Está escrito, probado, evaluado y con optimizaciones de rendimiento en curso, pero aún no está revisado.
+
+Digamos que x es la clave privada, P es xG así que P es la clave pública. Entonces tienes k que es el nonce. Y luego tienes R que es el nonce por G. Y m es el mensaje. La función Hash es H. Así que la fórmula Schnorr es que tienes una firma (R, s) que es (k * G, k - H(R || P || M) * x). Si no incluyes la clave pública en ese hash, no es una prueba de conocimiento. En bitcoin, la clave pública siempre está en el mensaje.
```

Comment:
nit: I know that this is not in the original transcript, but when writing equations I think it helps the reader when you use the `` formatting (like in <https://btctranscripts.com/advancing-bitcoin/2020/2020-02-06-antoine-riard-taproot-lightning/>) to distinguish them from normal text.